src/containers/profile-page/components/desktop/ProfilePage.tsx

```diff
@@ -159,6 +165,8 @@ const ProfilePage = ({
   loadMoreUserFeed,
   loadMoreArtistTracks,
   openCreatePlaylistModal,
+  // Collectibles
```

Comment:
what is this comment for?

Reply:
lol. nothing. part of my original thinking to avoid confusing myself. will remove